ntbs-service/Pages/LegacyNotifications/Index.cshtml.cs

```diff
@@ -54,6 +59,12 @@ private async Task<IActionResult> GetPage()
             NotificationBannerModel = await _legacySearchService.GetByIdAsync(LegacyNotificationId);
             if (NotificationBannerModel == null)
             {
+                var notificationWithLegacyIdExists = await _notificationRepository.NotificationWithLegacyIdExistsAsync(LegacyNotificationId);
+                if (notificationWithLegacyIdExists)
```

Comment:
I think this could just be inlined:
`if (await _notificaitonRepository.NotificationWithLegacyIdExistsAsync(LegacyNotificationId))`

But I don't have a big preference towards it... it does result in a slightly longer line.

Reply:
seems less readable when inlined